test/unit/cli.test.js

```diff
@@ -49,6 +49,7 @@ describe('cli parser', function () {
                         globalVar: [],
                         envVar: [],
                         folder: [],
+                        preserveOrder: false,
```

Comment:
Actually preserveOrder has a default value, so it would be present in the opts parameter and leaving it out here would make the test fail.
Kindly tell me a way out here.